editor/settings_config_dialog.cpp

```diff
@@ -493,6 +493,48 @@ EditorSettingsDialog::EditorSettingsDialog() {
 	updating = false;
 }
 
+<<<<<<< HEAD
+=======
```

Comment:
Hm.

Reply:
looks like bad merge?

Reply:
Whoops lol.

I haven't pushed the code from my latest changes btw (like the functionality shown in the shortcuts editor Gif)